source/modules/user-profile/user-profile.pug

```diff
@@ -0,0 +1,19 @@
+include ../header/header
+include ../footer/footer
+
+mixin user-profile()
+  main.user-profile
+    +header-dashboard('User Profile')
+    ul.user-profile__info-list
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+    div.user-profile__buttons
+      +button('Save', 'anchor').fr-btn-primary.mr-2
```

Comment:
not using mr, ml when you do task. 
These classes are only used by Backend developer